### ISSUE TITLE
Bug : Problem in signup

### ISSUE THREAD

**Issue by yube01** [ojasaklechayt/Oju-s-Blogs#29]
At the time of sign up when I enter strong password with more than 10 words it stills shows enter strong password

**Comment by ojasaklechayt** [comment]
@Pooja-Lohar25 what do you think about this ?

**Comment by Pooja-Lohar25** [comment]
@yube01 Thanks for the lookup, may i know the exact string that showed this error, because a string having just a length of more than 10 characters isn't a strong password. 

the code accepts the usual pattern of a strong password as follows :
password should have atleast
1.  1 uppercase 
2.  1 lowercase 
3.  1 digit
4.  1 special character 
5.  length = 8 or more

@ojasaklechayt 
We can add these guidelines in the UI if required.

**Comment by ojasaklechayt** [comment]
> @yube01 May i know the exact string that showed this error, because a string having just a length of more than 10 characters isn't a strong password.
> 
> the code accepts the usual pattern of a strong password as follows : password should have atleast
> 
> 1. 1 uppercase
> 2. 1 lowercase
> 3. 1 digit
> 4. 1 special character
>    5..  length = 8 or more
> 
> we can add these guidelines in the UI if required.

I think it would be great if we can have the instructions in the UI. But let's see what is the key @yube01 tried to give

**Comment by yube01** [comment]
Is there any kind of validation in password??

**Comment by Pooja-Lohar25** [comment]
> Is there any kind of validation in password??

yes it only accepts strong passwords as valid passwords or else produces error. 

@yube01 please mention what kind of string produced the error so that the bug can be fixed if it exists.

**Comment by yube01** [comment]
Oh sorry I didn't know about that validation

**Comment by ojasaklechayt** [comment]
@Pooja-Lohar25 I feel it would be great it we can update the conditions of strong password in the UI.

**Comment by rnavaneeth992** [comment]
I want to work on this issue

**Comment by ojasaklechayt** [comment]
> I want to work on this issue

go for it

**Comment by rnavaneeth992** [comment]
@ojasaklechayt One doubt this repo comes under toc winter right?

**Comment by ojasaklechayt** [comment]
> @ojasaklechayt One doubt this repo comes under toc winter right?

Yes

**Comment by rnavaneeth992** [comment]
@ojasaklechayt I created PR 
Can you check and update
